spec/contact_fixer_spec.rb

```diff
@@ -78,4 +78,45 @@
       end
     end
   end
+  describe '.update_connections_phone_numbers' do
+    before(:each) do
+      @replacement_pattern = '123'
+      @contact_number = "0118-999-881-999-119-725-3"
```

Comment:
Reference to https://theitcrowd.fandom.com/wiki/0118_999_881_99_9119_7253

Reply:
Hahaha, didn't see that episode yet